azext_iot/operations/hub.py

```diff
@@ -181,6 +181,114 @@ def iot_device_delete(cmd, device_id, hub_name=None, resource_group_name=None, l
         raise CLIError(unpack_msrest_error(e))
 
 
+def iot_device_get_parent(cmd, device_id, hub_name=None, resource_group_name=None, login=None):
+    target = get_iot_hub_connection_string(cmd, hub_name, resource_group_name, login=login)
+    child_device = _iot_device_show(target, device_id)
+    if child_device['capabilities']['iotEdge']:
+        raise CLIError('The device should be non-Edge device')
+    if 'deviceScope' not in child_device:
+        raise CLIError('The device doesn\'t have any parent set')
```

Comment:
Wording here is quite confusing. 
Maybe change it to 'Device <id> doesn't support parent device functionality'